src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.targets

```diff
@@ -494,6 +494,8 @@ Copyright (C) 2012 Xamarin Inc. All rights reserved.
       ToolPath="$(MonoAndroidToolsDirectory)"
       ToolExe="$(BindingsGeneratorToolExe)"
       UseShortFileNames="$(UseShortGeneratorFileNames)"
+      LangVersion="$(LangVersion)"
+      EnableInterfaceMembersPreview="$(EnableInterfaceMembersPreview)"
```

Comment:
Do we need to document this yet? https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/BuildProcess.md

Reply:
Good catch, I forgot about documentation.

Reply:
@jpobst: please update this section: https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/BuildProcess.md#binding-project-build-properties

To document the new `$(EnableInterfaceMembersPreview)` property.

Additionally, convention is that public properties begin with `Android`.

The `Preview` suffix is...disturbing?  What's the longevity for this property?

Should this instead be a `$(_EnableInterfaceMembers)` property?  A `_` prefix is "private"/"subject to change without notice" designator, so maybe that's what we should use?

Reply:
The intention is to explicitly denote that this field is temporary and not something we are going to support for the rest of time.  I am fine with a "private" property if that conveys the same idea.  :)